README.md

```diff
@@ -83,6 +83,14 @@ Remove the previous user.
 
 Setup docker and docker-compose in your machine (https://docs.docker.com/mac/ and https://docs.docker.com/compose/install/)
 
+If you already have a mysql database set up, you can try out Laika simply by running
+
+```
+docker run medigo/laika:latest laika run --mysql-host='<MYSQL_HOST>' --mysql-username='<MYSQL_USERNAME>' --mysql-password='<MYSQL_PASSWORD>'
+```
+
+Otherwise, follow the steps below.
+
 Clone the Laika project, open Docker Quickstart Terminal and go to the project directory.
 
 Create a .env file with a username and password
```

Comment:
Maybe you could describe how to put the username and password

I would say something like

- Create a *.env* file to set up a root user:

    ```
    LAIKA_ROOT_USERNAME=your_root_username
    LAIKA_ROOT_PASSWORD=your_root_password
    ```